### DISCUSSION TITLE
Failed to create nuxt3 project with `npx nuxi init nuxt-app`

### DISCUSSION THREAD

**Discussion by valcosmos** [nuxt/nuxt#16763] [Questions]
Today, I tried to create a new nuxt3 project, but I encountered the following problem, I spent some time trying to solve it using a search engine, but couldn't find a solution.

Does anyone know how to solve this situation please? Any reply will be greatly appreciated.

This happens in `MacOS 12.6` `node v16.17.0` `npm v8.15.0`.

```
❯ npx nuxi init nuxt-app
Nuxi 3.0.0-rc.10                                                                                              10:53:19

 ERROR  Failed to download template from registry: request to https://raw.githubusercontent.com/nuxt/starter/templates/templates/v3.json failed, reason: connect ECONNREFUSED 0.0.0.0:443

  at node_modules/nuxi/dist/chunks/init.mjs:13248:11
  at processTicksAndRejections (node:internal/process/task_queues:96:5)
  at async downloadTemplate (node_modules/nuxi/dist/chunks/init.mjs:13247:20)
  at async Object.invoke (node_modules/nuxi/dist/chunks/init.mjs:13315:15)
  at async _main (node_modules/nuxi/dist/cli.mjs:50:20)
```

**Comment by sevillaarvin** [comment]
Hi, try manually setting the registry: https://stackoverflow.com/a/45877279

Also check your `/etc/hosts`

**Reply by valcosmos** [reply]
Hello, thank u so much, i tried this solution, but it still doesn't work.😭

<img width="845" alt="image" src="https://user-images.githubusercontent.com/53572196/192071412-aaa1b275-bb79-47af-8034-4a1bb4b775b1.png">

**Reply by sevillaarvin** [reply]
Huh that is weird. Can you try curling it to see if it is a network issue? The response should be something like:

```
└──╼ $ curl https://raw.githubusercontent.com/nuxt/starter/templates/templates/v3.json
{
  "name": "v3",
  "defaultDir": "nuxt-app",
  "url": "https://v3.nuxtjs.org",
  "tar": "https://codeload.github.com/nuxt/starter/tar.gz/refs/heads/v3"
}
```

If the network is fine, try cleaning the cache with `npm cache clean --force`.

If that doesn't work, try updating or reinstalling `node`.

**Reply by echo9z** [reply]
Hanging proxy such as vpn

**Reply by valcosmos** [reply]
Thanks u so much for u help 😊

I tried `npm cache clean --force` and reinstalling node, It doesn't work in my machine. And I can get this template in terminal, but `npx nuxi init nuxt-app` still doesn't work.😭

<img width="850" alt="image" src="https://user-images.githubusercontent.com/53572196/192145825-c0aedf27-a0ca-42fa-9900-0de44a4241b7.png">

**Reply by sevillaarvin** [reply]
:scream: 

Hmm, I see you are in China, how about setting a mirror registry?

`npm config set registry http://r.cnpmjs.org`
or maybe
`npm config set registry https://registry.npm.taobao.org/`
or european?
`npm config set registry http://registry.npmjs.eu`

https://stackoverflow.com/a/27216097

**Reply by valcosmos** [reply]
Thanks u so much, bro. 👍

 I tried `npx nuxi init nuxt-app` on another device without any problem😂,  that's so weird, so I think it may be a problem with my device or the network.😭
